cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,124 @@
+Cypress.on('uncaught:exception', (_err) => {
+  return false;
+});
+
+let user;
 
 describe('Web Tables page', () => {
-  it('', () => {
+  before(() => {
+    // eslint-disable-next-line arrow-parens
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('Should check pagination and selector of rows.', () => {
+    cy.get('.-previous > .-btn').should('contain', 'Previous');
+    cy.get('.-next > .-btn').should('contain', 'Next');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+    cy.get('select').select('5 rows');
+    cy.get('select').should('contain', '5 rows');
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.contains('Next').click();
+  });
+
+  it('Create a new user.', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.get(':nth-child(4) > .rt-tr > :nth-child(1)').should('exist');
+  });
+
+  it('Delete user.', () => {
+    cy.get('#delete-record-1').click();
+    cy.get(':nth-child(3) > .rt-tr > :nth-child(1)').should('have.value', '');
+  });
+
+  it('Delete all users.', () => {
+    cy.get('#delete-record-1').click();
+    cy.get('#delete-record-2').click();
+    cy.get('#delete-record-3').click();
+    cy.get(':nth-child(1) > .rt-tr > :nth-child(1)').should('have.value', '');
```

Comment:
in this test you need to check text in the table, that "no rows found" after deletion

Reply:
Changes in commit https://github.com/mate-academy/qa_cypress_e2e_web_tables/pull/188/commits/a38dbac156617e5fbc07bf1a11770c0082a88050